module-2/backend/week_one.md

```diff
@@ -4,29 +4,65 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
+1. List the five common HTTP verbs and what the purpose is of each verb.  
+-GET - gets something  
+-POST -change/update something  
+-PUT -create something
+-DELETE -delete something
+-PATCH - alias to put?
+2. What is Sinatra?  
+Sinatra is a Ruby framework that sets up functionality to create basic web pages.  
+4. What is MVC?  
+Model View Controller  
+5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?  
+This allows programmers to look at and use each others code with ease and understanding.   
+6. What types of variables are accessible in our view templates without explicitly passing them?  
+Instance variables
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
-  get '/horses' do
+  get '/horses' do  
+  
     erb :index
   end
-  ```
-
+  ```  
+@count = 1
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
-9. What's the purpose of ERB?
-10. Why do I need a development AND test database?
-11. What's responsive design?
-12. What is CRUD and why is it important?
-13. What does HTTP stand for? 
+name = "Mr. Ed"
+erb :index, locals {name} (possibly something else with name?)

```

Comment:
Close. This is still Ruby so think of this as passing two arguments to the `erb` method - `:index`, and the locals - `locals: {name: name}`. The first name in the locals hash is the name of the variable as you're going to use it in the view, while the second is the name of the variable that you're passing it (which you set using `name = "Mr. Ed"`.